src/hb-subset.cc

```diff
@@ -121,8 +128,8 @@ _hb_subset_face_data_create (void)
   return data;
 }
 
-static void
-_hb_subset_face_data_destroy (void *user_data)
+void
+hb_subset_face_data_destroy (void *user_data)
```

Comment:
I don't understand. This is internal. What are you doing?

Reply:
Fixed.